src/tracing/errors.ts

```diff
@@ -9,9 +9,16 @@ declare global {
 }
 
 export function beforeSend(event: ErrorEvent, hint: EventHint) {
```

Comment:
This should retain the typing that was previously used for `filterKnownErrors`. I also think it should keep the comment, and that the new comment on L12 is unnecessary with the doc-comment restored.